src/components/form/HdTextarea.vue

```diff
@@ -80,6 +90,10 @@ export default {
       type: String,
       default: '100px',
     },
+    maxlength: {
+      type: [String, Number],
+      default: '',
```

Comment:
Why do you allow `String`? To be able to hide the maxlength indicator? You're using `Boolean(maxlength)` which will resolve to `false` even if the maxlength is `0`. So no need to allow `String` as a type.

But I would suggest to use `Number.POSITIVE_INFINITY` as a default value, and have a computed property `isMaxlengthIndicatorVisible` that looks something like this:
```js
isMaxlengthIndicatorVisible() {
 return this.maxlength < Number.POSITIVE_INFINITY;
}
```

Reply:
Got your point; thanks for the idea; here are the changes: https://github.com/homeday-de/homeday-blocks/pull/815/commits/504d02ceac39fc5ee2fdceb9ef2a76471a98a41c

> Why do you allow String?
I was allowing String to avoid console errors but can't see the errors now. There might be a case where we pass a number as a string value from a JSON object through the component directly. The reason was to avoid the console errors in these specific cases, but I can't produce them atm.